ical.md

```diff
@@ -0,0 +1,16 @@
+# ical
```

Comment:
move this to the `pages.de/common` directory

Reply:
@navarroaxel, the contributor has already added ical's German translation. It was accidentally merged to main instead of pages.de. Which we fixed in a PR today. 
I think PRs author is working on the same branch.